src/components/navigation/navigation.component.jsx

```diff
@@ -1,13 +1,32 @@
 import React from "react";
-//Styles
-import "./navigation.styles.scss";
-const Navigation = () => {
-    return (
-        <nav className="navigation">
-            <span>Projects</span>
-            <span>Bullet Journal</span>
-        </nav>
-    );
-};
+// Components
+import NavigationDesktop from "../navigation-desktop/navigation-desktop.component.jsx";
+import NavigationMobile from "../navigation-mobile/navigation-mobile.component.jsx";
+
+class Navigation extends React.Component {
+    constructor() {
```

Comment:
There is also a good practice to put 'props' as constructor argument and then pass it in 'super' method.

Reply:
True, forgot about it because Navigation wasn't meant to be a class component at first.